unittests/testeventsconstants.py

```diff
@@ -175,59 +156,6 @@
     "version": "CycloneDx 1.3 XML Default Version",
 }
 
-EVENT_ATTRS_SBOM = {
-    "operation": "Record",
-    "behaviour": "RecordEvidence",
-    "timestamp_declared": "2019-11-27T14:44:19Z",
-    "principal_declared": PRINCIPAL_DECLARED,
-    "event_attributes": {
-        "arc_description": "event description",
-    },
-    "sbom": {
-        "filename": "gen1.xml",
-        "content_type": "text/xml",
-        "confirm": True,
-        "params": {
-            "privacy": "PRIVATE",
-        },
-    },
-}
-REQUEST_WITH_SBOM = {
```

Comment:
we still want these don't we?

Reply:
no - this is only used when uploading sbom via sbom hub